shardingsphere-agent/shardingsphere-agent-api/src/main/java/org/apache/shardingsphere/agent/api/advice/ClassStaticMethodAroundAdvice.java

```diff
@@ -26,6 +26,15 @@
  */
 public interface ClassStaticMethodAroundAdvice {
     
+    /**
+     * Check if skip the enable checking.
+     *
+     * @return skip or not
+     */
+    default boolean skipEnableCheck() {
```

Comment:
What's mean of skipEnableCheck?

Reply:
If the value is true the interceptor will ignore the check process and call the advice directly